app/models/manageiq/providers/azure/cloud_manager/orchestration_service_option_converter.rb

```diff
@@ -1,9 +1,11 @@
 module ManageIQ::Providers
   class Azure::CloudManager::OrchestrationServiceOptionConverter < ::ServiceOrchestration::OptionConverter
     def stack_create_options
+      resource_group = @dialog_options['dialog_resource_group']
+      resource_group = @dialog_options['dialog_new_resource_group'] if resource_group.blank?
       {
         :parameters     => stack_parameters,
-        :resource_group => @dialog_options['dialog_resource_group'] || @dialog_options['dialog_new_resource_group'],
+        :resource_group => resource_group,
```

Comment:
Use `.presence` which would only require modifying the original line:
`:resource_group => @dialog_options['dialog_resource_group'].presense || @dialog_options['dialog_new_resource_group'],`